Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json

```diff
@@ -131,6 +133,8 @@
       "ny_ic_direct_call" : true,
       "ny_ic_dl_blue" : true,
       "ny_ic_download" : true,
+      "ny_ic_driver_addition_table2" : true,
+      "ny_ic_driver_gender_banner" : true,
```

Comment:
@Akash2172 Images are added in driver folder and jupay_assets.json is updated user folder directory , Please check